examples/EmbedApp.purs

```diff
@@ -19,19 +20,7 @@ attach = apply appSetup
 -- Pretend this is some external JS file and we want
 -- to attach our PS handlers into express app that
 -- already exists there.
```

Comment:
I guess this comment could be removed too since we do not need to pretend no more:)